tests/ext/check_memory_under_limit_default.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
```

Comment:
These tests are 🔝! We just gotta add all of them to the **package.xml** file. :)

Reply:
Everytime I added new file I thought, remember to add it to package.xml - remember to add it to package.xml. 
Then I forgot :D

Reply:
Haha! I'm the same way! I think to myself: this time I won't forget. Then I totally forget. 😆